services/aclarai-ui/aclarai_ui/config_panel.py

```diff
@@ -264,6 +264,16 @@ def load_current_config() -> Tuple[
         bool,
         bool,
         str,
+        # Concept highlights configuration
+        str,   # top_concepts_metric
+        int,   # top_concepts_count
+        float, # top_concepts_percent
+        str,   # top_concepts_target_file
+        int,   # trending_topics_window_days
+        int,   # trending_topics_count
+        float, # trending_topics_percent
+        int,   # trending_topics_min_mentions
+        str,   # trending_topics_target_file
     ]:
```

Comment:
The load_current_config function's return signature is a very large tuple. While this PR follows the existing pattern, the growing number of configuration options makes this approach fragile and hard to maintain. Refactor to use a dataclass or a dictionary for return values would improve readability and robustness.